PocketBeagle/.work-in-progress/Grove/Kit.md

```diff
@@ -73,33 +28,35 @@ Passphrase? *************
 Connected wifi_e8de27077de3_41483034303434393134_managed_psk
 connmanctl> quit
 ```
+we can use `ifconfig` to get IP of pocketbeagle
 
 # Lesson - 1. Control the Light
 
 ## Description:
 
-In this lesson, students will light up the RGB LED, and learn how to use Slide Potentiometer and Rotary Angle Sensor to change the light of RGB LED.
+In this lesson, students will light up the RGB LED and LCD, and learn how to use Slide Potentiometer and Rotary Angle Sensor to change the light of RGB LED.
```

Comment:
Thanks for incorporating my LCD updates. I believe it makes a good early experience to see content on the LCD where debug statements can be printed, even if the console is not available.